src/DynamoCore/Graph/Workspaces/WorkspaceModel.cs

```diff
@@ -268,6 +275,10 @@ protected virtual void OnConnectorAdded(ConnectorModel obj)
             RegisterConnector(obj);
             var handler = ConnectorAdded;
             if (handler != null) handler(obj);
+            if (workspaceLoaded)
```

Comment:
can you just add this note on github as a comment in the code. I think this needs to be documented.